dashboard/test/ui/features/applab.feature

```diff
@@ -1,9 +1,12 @@
 @eyes
 Feature: Applab Stuff
 
+@dashboard_db_access
 Scenario: Button shows up on top of canvas
+  Given I am on "http://learn.code.org/"
+  And I am a student
```

Comment:
AFAICT the step: `And I am on "http://studio.code.org/users/sign_in"` is not strictly necessary for login, seems to work and logins in other features omit it. It might actually make sense to collapse this idiom into a single compound sign-in step